cmd/newrelic-infra/newrelic-infra.go

```diff
@@ -270,7 +271,10 @@ func initializeAgentAndRun(c *config.Config, logFwCfg config.LogForward) error {
 		c.PluginInstanceDirs,
 		pluginSourceDirs,
 	)
-	integrationEmitter := emitter.NewIntegrationEmitter(agt, dmSender, ffManager)
+	icfg := identity.NewConfiguration()
+	icfg.Host = "staging-identity-api.newrelic.com"
+	identityClient := identity.NewAPIClient(icfg)
+	integrationEmitter := emitter.NewIntegrationEmitter(agt, dmSender, identityClient.DefaultApi, ffManager, c.License, userAgent)
```

Comment:
This is temp code that will be removed but needed something to be able to test this with.